src/reference/02-DetailTopics/05-Plugins-and-Best-Practices/04-GitHub-Actions-with-sbt.md

```diff
@@ -44,20 +46,30 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@v4
-    - name: Setup JDK
-      uses: actions/setup-java@v3
-      with:
-        distribution: temurin
-        java-version: 8
-    - name: Build and Test
-      run: sbt -v +test
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 11
```

Comment:
I think it is good to specify `sbt` caching: https://github.com/actions/setup-java#caching-packages-dependencies
It's use like this here, e.g.: https://github.com/sbt/sbt-native-packager/blob/main/.github/workflows/validate-pr.yml#L19-L24

Reply:
There is already a [section on caching](https://www.scala-sbt.org/1.x/docs/GitHub-Actions-with-sbt.html#Caching) on that page that talks about adding that exact option.

The whole page could do with a full rewrite to bring it up date but that can be done on the new docs site. For now I wanted to do a quick update so that the examples work.

Reply:
Sorry, my bad I was only looking at the diff 🙈